backend/debug_cloudkit_simple.py

```
@@ -0,0 +1,104 @@
+            )
+        print("✅ Private key loaded")
+    except Exception as e:
+        print(f"❌ Failed to load private key: {e}")
+        return False
+    
+    # Test zone listing (simplest operation)
+    print(f"\n🌐 Testing zone listing...")
+    try:
+        iso_date = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
+        endpoint = "zones/list"
+        method = "POST"
```

Ruff (0.11.9)

Local variable `method` is assigned to but never used

Remove assignment to unused variable `method`

(F841)